### PR TITLE
CLOUDP-248476: Apply SDLC checklist feedback

### PR DESCRIPTION
**Pull request by josvazg**
NOTE: Upload to Silk is pending (See CLOUDP-239190), once that is ready we will revisit the SBOMs linked to be the augmented ones.

### All Submissions:

* [X] Have you signed our [CLA](https://www.mongodb.com/legal/contributor-agreement)?

### CODE REVIEW

**Comment by josvazg** [comment]
Will review comments on https://github.com/10gen/ops-manager-kubernetes/pull/3569 before coming back to this

**Review by igor-karpukhin — APPROVED** [review]

File: docs/releases/sdlc-compliance.template.md [hunk 1]
```diff
@@ -39,10 +39,6 @@ Overview:
...
-1. Internal processes are used to ensure CVEs are identified and mitigated within SLAs.
+- Internal processes are used to ensure CVEs are identified and mitigated within SLAs.
 
-2. The Dependency document does not specify third party OSS CVEs fixed by the release and the date we discovered them.
-
-3. There is no CycloneDX field for original/modified CVSS scor or discovery date. The `x-` prefix indicates this.
-
-3. Assumption: We can include the SBOMs as links to read-only files on S3. The links can be included as metadata or text file links in release artifacts e.g. as labels on OCI containers.
\ No newline at end of file
+- All Operator images are signed by MongoDB, with signatures stored at `docker.io/mongodb/signatures`.
```
Comment (slaskawi) [inline docs/releases/sdlc-compliance.template.md:44]: We probably need to document a way to verify the signatures (a set of commands typically ran by a user) and put it somewhere in the project docs. You can probably copy-paste it from DOCSP-39646.

**Review by slaskawi — CHANGES_REQUESTED** [review]
Overall looks good the only bit that is missing and needs to be corrected is the signing verification manual. Let me know if you'd like to address this in a separate PR, so that I'll approve this one.

File: docs/releases/sdlc-compliance.template.md (hunk 1 above)
Comment (josvazg) [reply docs/releases/sdlc-compliance.template.md:44]: We have that already here, check out line 28, which links to:
[docs/dev/signed-images.md](https://github.com/mongodb/mongodb-atlas-kubernetes/blob/main/docs/dev/signed-images.md#image-signature-verification)

**Comment by josvazg** [comment]
> Overall looks good the only bit that is missing and needs to be corrected is the signing verification manual. Let me know if you'd like to address this in a separate PR, so that I'll approve this one.

It is done, see line 28 and [/docs/dev/signed-images.md](https://github.com/mongodb/mongodb-atlas-kubernetes/blob/main/docs/dev/signed-images.md#image-signature-verification)

File: docs/releases/sdlc-compliance.template.md (hunk 1 above)
Comment (slaskawi) [reply docs/releases/sdlc-compliance.template.md:44]: Missed it. Approving! Thanks!

**Review by slaskawi — APPROVED** [review]

**Pull request merged**